src/calculateRentalCost.js

```diff
@@ -28,7 +28,19 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const totalPrice = days * 40;
+  const minDisc = 20;
+  const maxDisc = 50;
```

Comment:
'disc' - it's not a meaningful name. Use 'discount' instead. Somebody else will not understand what means 'disc' in your code.